README.md

```diff
@@ -0,0 +1,173 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Budget app project <a name="about-project"></a>
+<br>
+This Ruby on Rails project is about a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.<br>
+<br>
+
+## 🛠 Built With <a name="built-with"></a><br>
+
+### Tech Stack
+
+This project is built using the following technologies:
+
+<details>
+  <summary>Programming languages</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<br>
+
+### Key Features <a name="key-features"></a>
+
+<br>
+
+#### The app allows the user to:
+
+- **Register and log in, so that the data is private to them.**
+- **Introduce new transactions associated with a category.**
+- **See the money spent on each category**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo & Video presentation<a name="live-demo"></a> <a name='video'></a>
+<br>
+
+- [Check the live demo, mobile version](https://budget-app-ee56.onrender.com/)
+- [Check quick demonstration of the app video by clicking the link](https://www.loom.com/share/857a8ae3f7164af298df568067e019ef)
```

Comment:
- [ ] Amazing job working on the video presentation but I think it would be amazing if you mention a piece of code you are proud of since it is one of the requirements you should also highlight some interesting piece of code or something you built that you are very proud of. ⭐🚀💯

![image](https://github.com/pushkar1995/budget-app/assets/58553711/b8c80fa1-38ac-4a19-92e0-5e4297f35285)

Reply:
Well resolved!